app/javascript/component/app.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import Greetings from "./greetings";
+
+function App() {
+  return (
+    <BrowserRouter>
+      <Routes>
+       <Route path="/" element={<Greetings />}></Route>
+      </Routes>
+    </BrowserRouter>
+  );
+}
+
+export default App;
```

Comment:
- [ ] Hi @emarkees, the two apps are not connecting together successfully. I ran the app after creating random greetings from the console but unfortunately no random greetings are displayed as you can see on the screenshot below. I think you'll required to fix the issue.
![Screenshot from 2023-10-05 19-07-00](https://github.com/emarkees/hello-rails-react/assets/63087491/56961a4e-c259-467e-8ed1-b1c95342dad3)